src/main/java/no/uib/inf112/core/map/tile/api/MovableTile.java

```diff
@@ -15,6 +17,14 @@
      */
     void move(int dx, int dy, int maxTime);
 
+    /**
+     * The tile does a move in a given direction
+     *
+     * @param dir direction to move in
+     * @return true if tile was able to move
+     */
+    boolean moveOneStep(Direction dir);
```

Comment:
Maybe renamed to just `move`? The one step part should be in the javadoc I think

Reply:
I did this change, but i started doubting it. I like this name better since I'm using it in the move method in Robot, i thought about using this as a private method in the robot class, but its needed to be able to push. But we can keep it named move(Direction)